compiler/noirc_frontend/src/tests/expressions.rs

```diff
@@ -226,3 +226,29 @@ fn must_use() {
     "#;
     check_errors(src);
 }
+
+#[test]
+fn duplicated_mut_in_basic_let_pattern() {
+    let src = r#"
+    fn main() {
+        let mut mut _x = 1;
+                ^^^ Expected a pattern but found 'mut'
+                ^^^ Expected a '=' but found 'mut'
+                ^^^ Expected a statement but found 'mut'
+            ^^^ Expected a ; after `let` statement
+    }
+    "#;
+    check_errors(src);
+}
```

Comment:
This test looks unintentional - it doesn't have to do with tuples.
It does look like the parser recovery is quite bad in this case.